pkg/datasource/handler.go

```diff
@@ -26,6 +27,8 @@ import (
 const (
 	RetryInterval = 1 * time.Second
 	RetryCount    = 60
+
+	StorageNetworkInterface = "lhnet1"
```

Comment:
How do know the network interface name here?

Reply:
It's set by Longhorn manager.
```
StorageNetworkInterface = "lhnet1"
```

Reply:
ref: https://github.com/longhorn/longhorn-manager/pull/1212/files#diff-f8d2bd35e1a8c9b86c8c4448dde00bc5480652f85ed0630efc012c3906d3a14fR134